### DISCUSSION TITLE
[BP#50] Privilégier les variables locales

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices#149] [Bonnes pratiques] [answered]
J'aimerais étendre l'exemple aux variables CSS aussi pour réduire les calculs de styles.

Sources : 

- [https://lisilinhart.info/posts/css-variables-performance/](https://lisilinhart.info/posts/css-variables-performance/)
- [https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations)

**Comment by tbroyer** [comment]
Ces liens commencent à dater un peu (2017 et 2019 respectivement), est-ce qu'ils sont toujours d'actualité ?

Plus récents, j'ai trouvés ceux-ci:

* https://blogs.igalia.com/jfernandez/2020/08/13/improving-css-custom-properties-performance/
* https://blog.okopress.dev/css-variables-performance-benchmark-2021-f56cd26ae9f5

Mais puisqu'on parle de la BP#50, je serais plutôt d'avis de la supprimer en fait…

**Comment by ACTLEM** [comment, marked as answer]
Note équipe projet: Suppression de cette BP dont les résultats ne sont plus prouvées.

Issue créée: https://github.com/cnumr/best-practices/issues/168
